ysh/expr_eval.py

```diff
@@ -975,34 +975,26 @@ def _EvalSubscript(self, node):
                     if case2(value_e.Slice):
                         index = cast(value.Slice, index)
 
-                        if mylib.PYTHON:
-                            # stride not supported in mycpp
-
-                            step = index.step.i if index.step else 1
-
-                            try:
-                                if index.lower and index.upper:
-                                    return value.List(
-                                        obj.items[index.lower.i:index.upper.
-                                                  i:step])
+                        try:
+                            if index.lower and index.upper:
+                                return value.List(
+                                    obj.items[index.lower.i:index.upper.i])
```

Comment:
Mostly a tangential style change, but this function could be shortened now to:

```
lower = index.lower.i if index.lower else 0
upper = index.upper.i if index.upper else len(obj.items)
```

And then a single slice operation, instead of 4 cases